src/libraries/AccountAccessParser.sol

```diff
@@ -258,7 +259,11 @@ library AccountAccessParser {
     ///              timestamp, since some timestamps are packed into slots with other data.
     ///   4. The hash to return is computed as `keccak256(abi.encode(normalizedArray))`.
     /// @return bytes32 hash of the normalized state diff
-    function normalizedStateDiffHash(VmSafe.AccountAccess[] memory _accountAccesses) internal view returns (bytes32) {
+    function normalizedStateDiffHash(VmSafe.AccountAccess[] memory _accountAccesses, address multisig, bytes32 txHash)
```

Comment:
Is `multisig` supposed to be the parent multisig or child multisig? would be great to have a more clear variable name there (and also prefix with leading underscores per our regular convention)
```suggestion
    function normalizedStateDiffHash(VmSafe.AccountAccess[] memory _accountAccesses, address _multisig, bytes32 _txHash)
```

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/921/commits/c21496c65edf407d563cdfa08a32dcf028f636c6